capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -81,10 +81,12 @@ class ClubPage extends Component {
       alert('Assignment creation failed. You do not own this club.');
       return;
     }
+    const dueDate = document.getElementById('due-date').value;
     let data = {
       'clubID': this.state.club.id,
       'text': e.target[0].value,
-      'whenCreated': (new Date()).toUTCString()
+      'whenCreated': (new Date()).toUTCString(), 
```

Comment:
nit: You don't need the quotes on the left side of this

Reply:
Will fix!